msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/errors/MFAErrors.kt

```diff
@@ -5,24 +5,47 @@ import com.microsoft.identity.nativeauth.statemachine.results.MFARequiredResult
 import com.microsoft.identity.nativeauth.statemachine.results.MFASubmitChallengeResult
 
 /**
- * MFA error. The user should use the utility methods of this class
- * to identify and handle the error. This error is produced by the various
- * MFA operations.
+ * MFA error. Use the utility methods of this class
```

Comment:
Maybe this should be "MFARequestChallengeError"
```suggestion
 * MFA RequestChallengeError. Use the utility methods of this class
```

Same comment applies also for other MFA error classes